dip-0011.md

```diff
@@ -52,7 +52,7 @@ in DIP0012.
 
 Cryptographic identities have long drawn substantial attention in the blockchain
 space. Various solutions and iterations have been proposed inside of Dash's
-development circle over the years. In this document we propose a novel approach
+development circle over the years. This document proposes a novel approach
```

Comment:
I actually prefer the first style more :)

Reply:
@thephez ?

Reply:
Generally I don't have strong feelings on this one as there are varying opinions on which is better (plus it's not too unusual to see first person in BIPs). You guys can continue discussing if you like, but I will probably just merge whatever is here when all other items are resolved :slightly_smiling_face: 

BIP "we" count for reference:
``` sh
~/bips$ grep -Hrn 'we ' *.mediawiki | wc -l
219
```